doc/modules/index.md

```diff
@@ -85,30 +95,65 @@ The following Lua modules are enabled in the default configuration (but may requ
 - [spf.html](spf.html) - perform SPF checks
 - [trie](trie.html) - uses suffix trie for extra-fast patterns lookup in messages. (requires configuration)
-- [whitelist](whitelist.html) - provides a flexible way to whitelist (or blacklist) messages based on SPF/DKIM/DMARC combinations
 - [url_redirector](url_redirector.html) - dereferences redirects (requires Redis configuration)
+- [whitelist](whitelist.html) - provides a flexible way to whitelist (or blacklist) messages based on SPF/DKIM/DMARC combinations
+
+The following modules have been removed or deprecated:
+
+- [emails](emails.html) - extract email addresses from a message and checks them against DNS blacklists. (requires configuration)
+  - Deprecated, use [rbl](rbl.html).
+- [ip_score](ip_score.html) - dynamically scores sender reputation (requires Redis).
+  - Removed as of v2.0 and replaced by the [reputation module](reputation.html). The existing configuration
+in older installations is automatically converted by an Rspamd upgrade.
 
 The following modules are explicitly disabled in the default configuration, set `enabled = true` in `local.d/${MODULE_NAME}.conf` to enable them:
 
 - [mx_check](mx_check.html) - checks if sending domain has a connectable MX (requires Redis)
+- **p0f** - see below
+- [spamtrap](spamtrap.html) - catch spam trapped emails or even a catch-all domain. Intended for learning Bayes spam. (requires configuration)
 
-The following modules are explicitly disabled and are experimental, so you need to set `enabled = true` in `local.d/${MODULE_NAME}.conf` **AND** to set the global option `enable_experimental = true` in `local.d/options.inc`:
+The following modules are explicitly disabled and are experimental, see below to Enable a module **AND** set the global option `enable_experimental = true` in `local.d/options.inc`:
 
 - [url_reputation](url_reputation.html) - assigns reputation to domains in URLs (requires Redis). Removed in Rspamd 2.0.
 - [url_tags](url_tags.html) - persists URL tags in Redis (requires Redis). Removed in Rspamd 2.0.
 
 Experimental modules are not recommended for production usage!
 
+The following modules require documentation:
+
+- **http_headers**
+- **p0f** - passive OS fingerprinting : [Wikipedia](https://en.wikipedia.org/wiki/P0f) / Rspamd [PR #3037](https://github.com/rspamd/rspamd/pull/3037)
+
+
 ## Disabling module
 
-To disable an entire module you can set `enabled = false;` in `/etc/rspamd/local.d/${MODULE_NAME}.conf`
+To disable an entire module set `enabled = false;` in `$LOCAL_CONFDIR/local.d/${MODULE_NAME}.conf`.
+
+## Enabling a disabled module
+
+To enable an entire module set `enabled = true;` in `$LOCAL_CONFDIR/local.d/${MODULE_NAME}.conf`.
+
+## Config file locations for modules
+
+Here are some guidelines regarding where module configurations are stored:
+
+- The Rspamd-provided configuration files are in `$LOCAL_CONFDIR/modules.d`.
+- Your overrides are set in `$LOCAL_CONFDIR/local.d`
```

Comment:
Probably it is better to call that `local additions` instead of `overrides` as there are real overrides.

Reply:
Yes, will fix this and another reference on line 156. Thanks.